changelog/30678.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:change
+ui: (Client Counts) The mount attribution and namespace attribution charts have been replaced in the overview tab with a table that breaks down client attribution according to month selection.
```

Comment:
Can we use the same language and formatting as backend changelog entries ([example](https://github.com/hashicorp/vault/blob/main/changelog/30071.txt#L2)) to denote which aspect of Vault this applies to? `ui/activity` instead of `ui: (Client Counts)`

Also a slight rewording suggestion to explain in a little more detail what the table functionality includes.

```
ui/activity: Replaces mount and namespace attribution charts with a table to allow sorting 
client count data by `namespace`, `mount_path`, `mount_type` or number of clients for 
a selected month. 
```